code/web/src/modules/styleSurvey/surveyChoices.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { APP_URL } from '../../setup/config/env'
+import { H3 } from '../../ui/typography'
+
+const SurveyChoices = ({ choices, handleChoice, choiceCategory }) => {
+  const handleChange = (event) => {
+    const choice = event.target.closest('span')
+    const selected = choice.getAttribute('aria-checked') === 'true'
+    handleChoice(choice, selected)
+    if (selected) {
+      choice.setAttribute('aria-checked', 'false')
+    } else {
+      choice.setAttribute('aria-checked', 'true')
+    }
+  }
```

Comment:
aria-checked is a helpful accessibility attribute that helps keep track of what's selected.